Userland/Libraries/LibJS/Runtime/RegExpConstructor.cpp

```diff
@@ -91,4 +198,84 @@ JS_DEFINE_NATIVE_FUNCTION(RegExpConstructor::symbol_species_getter)
     return vm.this_value();
 }
 
+// get RegExp.input https://github.com/tc39/proposal-regexp-legacy-features#get-regexpinput
```

Comment:
```suggestion
// get RegExp.input, https://github.com/tc39/proposal-regexp-legacy-features#get-regexpinput
```

to match the existing style - same everywhere below